modopt/tests/test_opt.py

```diff
@@ -627,6 +629,22 @@ def test_d_support_norm(self):
 
         npt.assert_raises(ValueError, proximity.KSupportNorm, 0.0, 0)
 
+    def test_group_lasso(self):
+        random_data = 3 * np.random.random(self.group_lasso.weights[0].shape)
+        random_data = np.tile(
+            random_data,
+            (self.group_lasso.weights.shape[0], 1, 1)
+        )
+        result_data = 2 * random_data - 3
+        result_data = (result_data * (result_data > 0).astype('int')) / 2
```

Comment:
To be consistent with the other tests, this should be defined in the class `setUp` and consequently unset in the `tearDown`.

Reply:
Cool. Fixing